src/ClientApp/src/components/BohrungForm.js

```diff
@@ -294,9 +327,16 @@ export default function BohrungForm(props) {
               label="X-Koordinate der Bohrung"
               type="number"
               variant="standard"
-              {...register("x_coordinate", { validate: validateXCoordinate })}
+              {...register("x_coordinate", {
+                validate: {
+                  range: (v) => validateXCoordinate(v) || "Die X-Koordinate muss zwischen 2590000 und 2646000 liegen",
+                  distance: (v) =>
+                    validateDistance(v) ||
+                    "Die Distanz der neu erstellten Bohrung darf nicht mehr als 200 m zu den bereits vorhandenen Bohrungen betragen",
```

Comment:
PP: Die Einheit ohne Abstand neben die Zahl schreiben
```suggestion
                    "Die Distanz der neu erstellten Bohrung darf nicht mehr als 200m zu den bereits vorhandenen Bohrungen betragen",
```